src/Content.tsx

```diff
@@ -1,76 +1,60 @@
 import React from 'react';
 import { createRoot } from 'react-dom/client';
-import { ProfessorPopup } from './ProfessorPopup';
+import ProfessorPopup from './ProfessorPopup';
```

Comment:
should be `import { ProfessorPopup } from './ProfessorPopup';` since it isn't a default export

lgtm afterwards, dont need to re-review